zeppelin-plugins/launcher/docker/src/main/java/org/apache/zeppelin/interpreter/launcher/DockerInterpreterProcess.java

```diff
@@ -162,6 +162,8 @@ public String getInterpreterSettingName() {
   public void start(String userName) throws IOException {
     docker = DefaultDockerClient.builder().uri(URI.create(DOCKER_HOST)).build();
 
+    removeExistContainer(containerName);
```

Comment:
Could you add comment here why need to remove exist container ?

Reply:
I mean adding comment above this line of code, otherwise others will confuse to see that why needs to removeExistContainer

Reply:
done.